### PR TITLE
Changes for reducing the Sonarqube issue log

### PR DESCRIPTION
**Pull request by iamianmullins**
Some small changes. Let me know your thoughts. Thanks Eric!

### CODE REVIEW

File: src/kms_sink.cpp [hunk 1]
```diff
@@ -108,9 +107,8 @@ int KMSSink::configure(const libcamera::CameraConfiguration& config) {
 
   const std::vector<DRM::Mode>& modes = connector_->modes();
 
-  int ret = configurePipeline(cfg.pixelFormat);
```
[the remark this answers is not in the document]
Comment (iamianmullins) [reply src/kms_sink.cpp:111]: Great, I've implemented your suggestion above and Sonarcloud appears to accept the change. Thanks for your feedback, resubmitting now

File: src/kms_sink.cpp [hunk 2]
```diff
@@ -199,8 +197,9 @@ int KMSSink::stop() {
   request.addProperty(plane_, "CRTC_ID", 0);
   request.addProperty(plane_, "FB_ID", 0);
 
-  int ret = request.commit(DRM::AtomicRequest::FlagAllowModeset);
```
[the remark this answers is not in the document]
Comment (iamianmullins) [reply src/kms_sink.cpp:202]: Same as above

File: src/kms_sink.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericcurtin) [reply src/kms_sink.cpp:111]: LGTM

File: src/kms_sink.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericcurtin) [reply src/kms_sink.cpp:202]: Could this be:

`if (int ret = request.commit(DRM::AtomicRequest::FlagAllowModeset); ret < 0) {`

with the less than operator please.

**Comment by ericcurtin** [comment]
Could we remove the .scannerwork folder and all it's files please?

**Pull request merged**